remote_attestation/java/src/com/google/oak/remote_attestation/crypto/KeyNegotiator.java

```diff
@@ -1,7 +1,6 @@
-package com.google.oak.functions.client;
+package com.google.oak.remote_attestation.crypto;
```

Comment:
pre-existing: this class has a field named `keyPair`. I think it can be `private final`.

Reply:
Done